src/pybind/mgr/dashboard/frontend/src/app/core/navigation/breadcrumbs/breadcrumbs.component.ts

```diff
@@ -138,4 +141,13 @@ export class BreadcrumbsComponent implements OnDestroy {
 
     return of(value as T);
   }
+
+  private getTitleFromCrumbs(crumbs: IBreadcrumb[]): string {
+    const currentLocation = crumbs
+      .map((crumb: IBreadcrumb) => {
+        return crumb.text.startsWith('http') ? '' : crumb.text;
```

Comment:
This is the exact kind of example where a comment would be awesome: not to explain what this does (which is obvious), but in what case the Dashboard breadcrumbs include `http` (which I'd find weird :-S).

Reply:
Thanks @epuertat. I had taken this from https://github.com/ceph/ceph/pull/41073#pullrequestreview-681863692 suggestion. Looks like this check was not needed after upgrading to angular 12. I have updated the code.